src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -222,7 +232,9 @@ class PairedResult extends GraphContent {
                 graph.config,
                 graph.svg.select(
                     `g[aria-describedby="region_${this.dataTarget.key}"]`
-                )
+                ),
+                this.dataTarget.yAxis,
+                !!utils.notEmpty(this.dataTarget.valueRegionSubset)
```

Comment:
nit: can you remove !! in here.

Reply:
Updated: https://github.com/SharathAlur/carbon-graphs/commit/788fde01aa11c505624e86e5a4ce9bc7da6d0813